spec/migrations/migration_logs_to_elk_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'spec_helper'
+
+RSpec.describe 'migration ccdb logs to elk', isolation: :truncation, type: :migration do
+  let(:db) { Sequel::Model.db }
+  let(:tmp_migrations_dir) { Dir.mktmpdir }
+
+  before do
+    long_time_migration = <<-MIGRATION
+      Sequel.migration do
+        up do
+          10.times do
+            VCAP::Migration.logging('Still Migrating')
+          end
+          VCAP::Migration.logging('migration finished')
+        end
+        down do
+        end
+      end
+    MIGRATION
+
+    migration_file = "#{tmp_migrations_dir}/001_test_for_long_time_migration.rb"
+    File.write(migration_file, long_time_migration)
+  end
+
+  after do
+    Sequel::Migrator.run(db, tmp_migrations_dir, allow_missing_migration_files: true)
+    FileUtils.rm_rf(tmp_migrations_dir)
+  end
+
+  it 'runs the up migration without errors' do
+    allow(VCAP::Migration).to receive(:logging)
```

Comment:
To test the changed method in `VCAP::Migration` I suggest the following:

1. Define a `logger` test variable: `let(:logger) { instance_double(Steno::Logger) }`
2. Allow `Steno.logger` to be called only with the correct name (and return the mock from above): `allow(Steno).to receive(:logger).with('cc.db.migrations').and_return(logger)`
3. Expect that the correct messages are sent to `logger.info`: `expect(logger).to receive(:info).with(...)`